bazaarvoice/emodb
# Emo-7571_fix_some_test

One test tiny fix
  
Feature coverage is additionally updated in our (for now internal) testing artefacts.
How to Test and Verify
Check out this PR
Start emodb on local
Open postman
Import "emodb_tests.postman_collection.json" as a collection
Import "sor-1-_table.postman_environment.json" as environment variables
Execute tests in postman
Profit
Expected result: All tests are successfully executed on local environment

Risk
Low
There are no changes to application code base

Required Testing
All newly added postman tests should be executed without failures.

Risk Summary
There are no risks to application, only tests are involved and it is about adding a new tests
